src/current/_includes/v23.1/sidebar-data/feature-overview.json

```diff
@@ -0,0 +1,82 @@
+{
+    "title": "Feature Overview",
+    "is_top_level": true,
+    "items": [{
+        "title": "CockroachDB Products",
+        "urls": [
+          "/${VERSION}/choose-a-deployment-option.html"
+        ]
+      },
+      {
+        "title": "CockroachDB Basics",
+        "items": [
+          {
+            "title": "Why CockroachDB?",
+            "urls": [
+              "/${VERSION}/why-cockroachdb.html"
+            ]
+          },
+          {
+            "title": "Replication & Rebalancing",
+            "urls": [
+              "/${VERSION}/demo-replication-and-rebalancing.html"
```

Comment:
This page needs a bigger and more detailed introductory section similar to what we do in the Serializable Transactions page. I think we can add to the existing page rather than trying to use an include, but totally up to you.

Reply:
I'll open an issue for filling these pages out more. I don't want to add too much to the scope of this PR

Reply:
That works. Thanks.